decentraland/unity-renderer
# fix: Lazy Loading fixes

## What does this PR change?
Fixes #3105 
Fixes #3106

### Fixes included
- [x] Fixed visibility of the JumpIn button at offline friends.
- [x] Improved the visibility of the loading spinner when loading pages on friends and direct conversations.
- [x] Refactor of the search behaviour on friends and conversations.
- [x] Fixed pagination issues when scrolling down friends and conversations.
- [x] Fixed the loading of the avatar images from some conversation entries that were not being displayed correctly.
- [x] Sometimes a conversation entry was duplicated.
- [x] Prioritized the display of online friends in the list

## How to test the changes?
1. Go to https://play.decentraland.org/?renderer-branch=fix/lazy-loading-fixes
2. Check that everything related to Chat & Friends is workins correctly.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
